examples/server_client.rs

```diff
@@ -8,8 +8,10 @@ use std::thread;
 
 const SERVER: &str = "127.0.0.1:12351";
 
+fn test() {}
```

Comment:
whats this for?

Reply:
I taught it looked pretty!, no just kidding, it was from an other PR I was planning to make. But it should be removed.